docs/using-the-jdbc-driver/using-plugins/UsingTheAuroraInitialConnectionStrategyPlugin.md

```diff
@@ -0,0 +1,28 @@
+# Aurora Initial Connection Strategy Plugin
+The AWS Advanced JDBC Driver allows the user to configure their initial connection strategy using this plugin. The initial connection strategy chooses which reader endpoint is used to replace an old reader endpoint during failover. With this plugin enabled, the initial connection to a reader cluster endpoint will be chosen based on the configured strategy. Users can also choose how often to retry and the maximum allowed time for finding the reader in these initial connections using the parameters. This allows for more control over choosing the reader. This plugin is used to replace the old endpoint with a current endpoint while DNS is still updating. When a user connects to a cluster endpoint, the actual instance for a new connection is resolved by DNS. During failover, the cluster elects another instance to be the writer. While DNS is updating, which can take up to 40-60 seconds, if a user tries to connect to the endpoint they may be connected to an old node. Changing the initial connection strategy would change which reader endpoint is chosen to replace the old reader endpoint in this scenario.
```

Comment:
```suggestion
The Aurora Initial Connection Strategy Plugin allows users to configure their initial connection strategy, and can also be used to obtain a connection more reliably by replacing an out of date endpoint if DNS is updating. When the Aurora Initial Connection Strategy Plugin attempts to retrieve a connection, it may retry the connection attempt if there is a failure. Users are able to configure how often to retry a connection and the maximum allowed time to obtain a connection using the connection parameters.

When this plugin is enabled, the initial connection to a reader cluster endpoint will be to the reader host chosen based on the configured strategy. The [initial connection strategy](../ReaderSelectionStrategies.md) specifies how the driver determines which available reader to connect to.

This plugin also helps retrieve connections more reliably by replacing an out of date endpoint if DNS is updating. When a user connects to a cluster endpoint, the actual instance for a new connection is resolved by DNS. During failover, the cluster elects another instance to be the writer. While DNS is updating, which can take up to 40-60 seconds, if a user tries to connect to the endpoint they may be connected to an old node. Changing the initial connection strategy would change which reader endpoint is chosen to replace the old reader endpoint in this scenario.
```
Maybe something like this?